integration_tests/integration/addRestrictedPatient.cy.ts

```diff
@@ -88,61 +89,65 @@ context('Add prisoner', () => {
     cy.signIn()
   })
 
-  it('should display the search for prisoner page from the homepage', () => {
-    const page = homepage.goTo()
+  it('should display the search for prisoner page from the HomePage', () => {
+    const page = Page.verifyOnPage(HomePage)
     page.addRestrictedPatient().click()
-    PrisonerSearchPage.verifyOnPage()
+    Page.verifyOnPage(PrisonerSearchPage)
   })
 
   it('Completes a add prisoner journey', () => {
     cy.visit('/add-restricted-patient/search-for-prisoner')
-    const prisonerSearchPage = PrisonerSearchPage.verifyOnPage()
-    const prisonerSearchPageForm = prisonerSearchPage.form()
+    const prisonerSearchPage = Page.verifyOnPage(PrisonerSearchPage)
 
-    prisonerSearchPageForm.searchTerm().type('A1234AA')
-    prisonerSearchPageForm.submit().click()
+    prisonerSearchPage.searchTerm().type('A1234AA')
+    prisonerSearchPage.submit().click()
 
     cy.location().should(loc => {
       expect(loc.pathname).to.eq('/add-restricted-patient/select-prisoner')
       expect(loc.search).to.eq('?searchTerm=A1234AA')
     })
 
-    const prisonerSelectPage = PrisonerSelectPage.verifyOnPage()
+    const prisonerSelectPage = Page.verifyOnPage(PrisonerSelectPage)
 
-    prisonerSelectPage.resultsTable().then($table => {
-      cy.get($table)
-        .find('tr')
-        .then($tableRows => {
-          cy.get($tableRows).its('length').should('eq', 2) // 1 result plus table header
+    prisonerSelectPage
+      .resultsTable()
+      .find('tr')
+      .should($tableRows => {
+        expect($tableRows).to.have.length(2) // 1 result plus table header
 
-          const offenders = Array.from($tableRows).map($row => toOffender($row.cells))
+        const offenders = Array.from($tableRows).map($row => toOffender($row.cells))
 
-          expect(offenders[1].name).to.contain('Smith, John')
-          expect(offenders[1].prisonerNumber).to.eq('A1234AA')
-          expect(offenders[1].locationDescription).to.eq('Outside - released from Doncaster')
-          expect(offenders[1].alerts).to.contain('Controlled unlock')
-          expect(offenders[1].addRestrictedPatientLink).to.contain('Add to restricted patients')
-        })
-    })
+        expect(offenders[1].name).to.contain('Smith, John')
+        expect(offenders[1].prisonerNumber).to.eq('A1234AA')
+        expect(offenders[1].locationDescription).to.eq('Outside - released from Doncaster')
+        expect(offenders[1].alerts).to.contain('Controlled unlock')
+        expect(offenders[1].addRestrictedPatientLink).to.contain('Add to restricted patients')
+      })
     prisonerSelectPage.addRestrictedPatientLink().click()
 
-    const selectHospitalPage = SelectHospitalPage.verifyOnPage('John Smith')
-    const selectHospitalPageForm = selectHospitalPage.form()
+    const selectHospitalPage = Page.verifyOnPageWithTitleParam(SelectHospitalPage, 'John Smith')
```

Comment:
this is slightly naff - that we end up calling different `verifyOnPage` methods, but is what the nomis sync dashboard does so have copied it from there